src/odra_toml.rs

```diff
@@ -47,4 +47,11 @@ impl OdraToml {
     pub fn has_contract(&self, contract_name: &str) -> bool {
         self.contracts.iter().any(|c| c.name == contract_name)
     }
+
+    /// Check if any contract in Odra.toml is a part of a module with given name
+    pub fn has_module(&self, module_name: &str) -> bool {
+        self.contracts
+            .iter()
+            .any(|c| c.fqn.split_terminator("::").any(|part| part == module_name))
```

Comment:
Shouldn't you check just the first `fqn` element?

Reply:
You're right